client/gutenberg/extensions/related-posts/style.scss

```diff
@@ -29,6 +33,7 @@
 	}
 
 	.wp-block-a8c-related-posts__preview-post-date {
-		font-style: italic;
+		color: $dark-gray-300;
+		font-size: $default-font-size;
```

Comment:
Just a thought; we can very well ship even if we decide to implement this later:

Gutenberg lets [themes pass font sizes to blocks](https://wordpress.org/gutenberg/handbook/extensibility/theme-support/#block-font-sizes). Should we follow that pattern for this block and use those classes instead of setting a fixed font size?

Reply:
We should.

Reply:
Good idea 👍 . 

That causes a small difference in the resulting font size by default (14px instead of 13px), but it seems like a good change towards consistency.

Addressed in https://github.com/Automattic/wp-calypso/pull/27456/commits/1da7f7df9a75b5a64ab1591712fb414ed49180ab

Reply:
Here's a screenshot of how the date looks now after the changes:

![](https://cldup.com/3W6Lv4Y1jy.png)